todo_app/app.py

```diff
@@ -1,15 +1,27 @@
-from flask import Flask
+from flask import Flask, render_template, request, redirect
 
 from todo_app.flask_config import Config
 
+from todo_app.data.session_items import get_items, add_item
+
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    #return 'Hello World!'
+#Code that fetches all the TODO items from session
+    items = get_items()
+    return render_template('index.html', items=items)
 
+@app.route('/addItem', methods=['POST','GET'])
+def formHandler():
```

Comment:
Python naming convention is snake_case for variable and function names, so `form_handler`.

Also formHandler is too generic. You will have more forms later that will need different functions.